community/lesstif/build

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh -e
+
+./configure \
+    --prefix=/usr \
+
+make
+make DESTDIR="$1" install
+
+rm -r "$1/usr/LessTif"
```

Comment:
Always add `-f` to `rm` to ensure that it doesn't prompt the user for confirmation (among other things).